### PR TITLE
In with Scala-3.0.0-RC1, out with 3.0.0-M2

### PR DESCRIPTION
**Pull request by rossabaker**

### CODE REVIEW

File: project/Dependencies.scala [hunk 1]
```diff
@@ -2,11 +2,29 @@ import sbt._
...
+      case "3.0.0-RC1" => "3.2.4"
+    }
+  final val scalatestPlusScalacheckVersion =
+    Def.map(scalaBinaryVersion) {
+      case "2.11"      => "3.2.3.0"
+      case "2.12"      => "3.2.4.0"
+      case "2.13"      => "3.2.4.0"
+      case "3.0.0-M3"  => "3.2.3.0"
+      case "3.0.0-RC1" => "3.2.4.0"
+    }
```
Comment (rossabaker) [inline project/Dependencies.scala:27]: This is a mess.  Scala 2.11 got dropped.  Unlike most libraries, only one version of Scala 3 is supported concurrently. 
 There's no scala.js support in the RC1 version.

We might want to consider porting the specs to MUnit, but I'm more concerned with unblocking people.  Maybe we can mop up the dottyJS support before 1.10.

**Review by rossabaker — COMMENTED** [review]
I propose releasing this as 1.10.0-M5.

**Comment by rossabaker** [comment]
This is all dependency bumps.  I'm going to go ahead and release the milestone to unblock downstream.

File: project/Dependencies.scala (hunk 1 above)
Comment (sarahgerweck) [reply project/Dependencies.scala:27]: Seems like a good idea if it's better supported.

Comment (rossabaker) [reply project/Dependencies.scala:27]: It's usually the first one released, and it has the full matrix from 2.11 through 3.0, which exactly matches our needs.  I don't like to chase fashion, but it really does seem like the best fit now.

**Pull request merged**